workflows/rnaseq.nf

```diff
@@ -191,11 +191,10 @@ workflow RNASEQ {
     .reads
     .map {
         meta, fastq ->
-            def meta_clone = meta.clone()
-            meta_clone.id = meta_clone.id.split('_')[0..-2].join('_')
-            [ meta_clone, fastq ]
+            without_technicalrep_number = meta.id - ~/_T\d+/
```

Comment:
Does this have to be a sooopper duper long name? What about `new_id` or something?

Reply:
Shortened